apps/voting/contracts/Voting.sol

```diff
@@ -235,6 +257,16 @@ contract Voting is IForwarder, AragonApp {
         return _canVote(_voteId, _voter);
     }
 
+    /**
+    * @notice Tells whether `_sender` can object to a vote #`_voteId` by voting 'no'
```

Comment:
@notice Tells whether `_sender` can participate in the objection phase of the vote #`_voteId`